### DISCUSSION TITLE
LinkAce deployed via Stack Portainer

### DISCUSSION THREAD

**Discussion by Tataukee** [Kovah/LinkAce#526] [Guides]
Hello guys ,

Here is a good news for those who are unconfrotable with CLI and struggling to install LinkAce through Portainer .
Luka Manestar from [Blackvoid](https://www.blackvoid.club/linkace-self-hosted-link-archive-platform/) did a wonderful job and managed to get the application deployed easily . An interesting article among others .
Bonne chance !  everyone .

**Comment by yamikeigo** [comment]
wish I would've seen this earlier. I was stumped on why the current guide wasn't working for me. kept stalling out at the gui setup page. So I just ended up installing it outside of Portainer in the /opt directory. I assume it has to do with the PHPAdmin container. I'll have to give this a shot later. Only because it annoys me having one dingleberry stack that I can't interact with via Portainer
